java/ql/src/semmle/code/java/dataflow/ParityAnalysis.qll

```diff
@@ -1,4 +1,6 @@
 /**
+ * DEPRECATED: semmle.code.java.dataflow.ModulusAnalysis instead.
```

Comment:
Missing word "use" before the library name.

Ditto further below.

Reply:
fixed